src/mongo/MongoScrapbook.ts

```diff
@@ -348,7 +345,7 @@ class FindMongoCommandsVisitor extends MongoVisitor<MongoCommand[]> {
 		}
 		let argumentContextArray: mongoParser.ArgumentContext[] = filterType(argumentsToken.children, mongoParser.ArgumentContext);
 
-		let functionMap = { "ObjectId": this.objectIdToObject, "ISODate": this.dateToObject, "Date": this.dateToObject };
```

Comment:
Why did this change? This doesn't seem related to https://github.com/microsoft/vscode-cosmosdb/issues/981. If you're fixing a different bug I would prefer a separate PR or at least mention that bug in the PR description.

Reply:
The changes there would depend on this PR (doing it independently complicates the code).  I chose to address the issue here. I mentioned it in the commit message.  Will add to PR description.